packages/ui/Scrollable/Scrollbar.js

```diff
@@ -14,6 +14,21 @@ const
 	minThumbSize = 18, // Size in pixels
 	thumbHidingDelay = 400; // in milliseconds
 
+function useHidingThumbJob (callback, timeout) {
+	const [state] = React.useState({});
+	state.job = state.job || new Job(callback, timeout);
```

Comment:
My recommendation is...
```
const hideThumbJob = useRef(new Job(callback, timeout));
```
It would be better to store the instance  in not `useState` but `useRef` because we don't change it.

Reply:
I don't think we are safe to assume that `callback` will not be changed ever. Also, `Job` related code may be refined later also later.

Reply:
Add)
When I tested, `callback` is not changed actually. Also, this hook is not so reusable nor helpful for the separation of concerns. So I  did put this back into Scrollbar with `useRef`.